crates/sui-config/src/node.rs

```diff
@@ -82,6 +86,10 @@ pub struct NodeConfig {
     pub genesis: Genesis,
 }
 
+fn default_gossip_degree() -> usize {
+    20
```

Comment:
This magic number seems strange. It seems to me that a default degree should be small enough (i.e. 4) that can cover all networks, or completely without a default to require a value that's dependent on the size of the network

Reply:
the number really depends on the committee size, so any number will be sort of "magic"
maybe we should remove the default value - but I suspect we will forget to tune this number before the dynamic re-genesis is in place. 

The reason for 20: the private testnet now has 20 validators, also partially I want to see how having a higher degree will impact the performance of the cluster

Reply:
If this number highly depend on the network size, then we probably shouldn't have a default there. If any, it would need to be a number that works for any network (i.e. 4, since it's the minimum size of the network)